app/src/main/java/com/compscieddy/meetinthemiddle/ChatFragment.java

```diff
@@ -162,24 +166,35 @@ public void onDataChange(DataSnapshot dataSnapshot) {
             } else {
               chatView.setIsSender(false);
             }
+             if (mChatsFirebaseAdapter.getItemCount() > 0) {
+              mEmptyChatView.setVisibility(View.INVISIBLE);
```

Comment:
Good job - but let's actually implement more formally like this: http://akbaribrahim.com/empty-view-for-androids-recyclerview/